scenes/game_logic/sequence_puzzle.gd

```diff
@@ -97,72 +68,38 @@ func _on_kicked(object: SequencePuzzleObject) -> void:
 		_debug("Didn't match")
 		for r: SequencePuzzleObject in _objects:
 			r.stop_hint()
-		if hint_levels.get(get_progress(), 0) >= wobble_hint_min_level:
-			hint_timer.start()
 		return
-
```

Comment:
Please restore this blank line, and all the subsequent blank lines that were removed.